### PR TITLE
Added a new endpoint for showing the session key of an encrypted message

### PR DESCRIPTION
**Pull request by ferglor**
Added the `/show-session-key` endpoint, which accepts a payload containing an encrypted message, and returns the session key for that encrypted message

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -368,3 +368,54 @@ $ curl \
...
+    --data @payload.json \
+    https://vault.example.com/v1/gpg/show-session-key/my-key
+```
+
+#### Sample Response
+
+```json
+{
+  "data": {
+    "sessionKey": "9:720D9B92D50D4F7C404C8C412BEB73B47E0A2FA2E822C13201A79D5A2694F9F5"
```
Comment (LeSuisse) [inline README.md]: Can we use `session_key` instead to keep the consistency with the others endpoints and the rest of the Vault API?

**Comment by LeSuisse** [comment]
Hello,

Thanks a lot for doing this! Code looks great, I'm going to do some functional testing and merge it!

File: README.md (hunk 1 above)
Comment (ferglor) [reply README.md]: sure thing! I'll update it now

**Comment by ferglor** [comment]
@LeSuisse updated!

**Comment by LeSuisse** [comment]
Thank you again for your work!

While testing I just found and fixed one small issue: since a message can be encrypted with multiple keys we can have multiple session packets so it should not be assumed that the first one is always the one we can decrypt.

**Pull request merged**